bedrock/firefox/templates/firefox/privacy/base.html

```diff
@@ -20,31 +20,7 @@
 {% endblock %}
 
 {% block sub_navigation %}
-<nav class="c-sub-navigation">
-  <div class="mzp-l-content">
-    <div class="c-sub-navigation-content">
-      <h2 class="c-sub-navigation-title is-summary">{{ ftl('sub-navigation-privacy', fallback='firefox-privacy-privacy') }}</h2>
-      <ul class="c-sub-navigation-list is-details is-closed">
-        <li class="c-sub-navigation-item">
-          <a href="{{ url('firefox.privacy.index') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Our Promise">{{ ftl('sub-navigation-our-promise', fallback='firefox-privacy-our-promise') }}</a>
-        </li>
-        <li class="c-sub-navigation-item">
-          <a href="{{ url('firefox.privacy.products') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Our Products">{{ ftl('sub-navigation-our-products', fallback='firefox-privacy-our-products') }}</a>
-        </li>
-      {% if ftl_has_messages('sub-navigation-little-book-of-privacy') %}
-        <li class="c-sub-navigation-item">
-          <a href="{{ url('firefox.privacy.book') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Little Book of Privacy">{{ ftl('sub-navigation-little-book-of-privacy') }}</a>
-        </li>
-      {% endif %}
-      {% if ftl_has_messages('sub-navigation-safe-passwords') %}
-        <li class="c-sub-navigation-item">
-          <a href="{{ url('firefox.privacy.passwords') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Safe Passwords">{{ ftl('sub-navigation-safe-passwords') }}</a>
-        </li>
-      {% endif %}
-      </ul>
-    </div>
-  </div>
-</nav>
+  {% include 'firefox/privacy/includes/sub-nav.html' %}
```

Comment:
On this file, in line 28, I see this block of code:
`{% block sub_nav %}{% endblock %}`
Should we remove it since you've already included the updated `side_nav `on this line?

Reply:
Good catch! Yes, this appears to be dead code leftover from https://github.com/mozilla/bedrock/pull/9650/files#diff-5568b6af1f54d3948e2e55fb396463696562149cf7dc1719bdca2d903c9a2743

Will remove.